JustSaying/JustSayingBus.cs

```diff
@@ -122,11 +124,13 @@ public void Start()
                     return;
                 }
 
+                _subscriberCancellationTokenSource = new CancellationTokenSource();

```

Comment:
Yeah, seems like we'd want to make `IDisposable` available in this class also, otherwise we're just moving the problem around.